platform-api/src/main/java/io/datacater/core/deployment/DeploymentEndpoint.java

```diff
@@ -133,6 +134,24 @@ public Uni<Response> getMetrics(
                 }));
   }
 
+  @GET
+  @Path("{uuid}/status")
+  @Produces(MediaType.APPLICATION_JSON)
```

Comment:
Do we need this annotation? Looks like we already defined it on the class level.

Reply:
You're right, the annotation is not needed